### PR TITLE
docs: suggest using image digests for security reasons

### PR DESCRIPTION
**Pull request by phisco**
Suggesting to use image digests instead of tags for security reasons.
This is due to the underlying implementation of go-containerregistry we use to pull images, which is not able to validate digests if not provided, see the [code](https://github.com/google/go-containerregistry/blob/5fe7f2e11795fcfd5a7c7d3a550c4e668a9a9789/pkg/v1/remote/fetcher.go#L172-L180) for more details.
I didn't specifically point out the details, as I feel it's kind of well known in the field that specifying digests is a good practice and a requirement in highly regulated and security-aware environments.

### CODE REVIEW

File: content/master/concepts/packages.md [hunk 1]
```diff
@@ -332,6 +332,10 @@ to avoid having to manually activate new versions. However, each user should
 consider their specific environment and choose a combination that makes sense
 for them.
 
+For security reasons, it's suggested using image digests instead or alongside
+tags (`vx.y.z@sha256:...`), to ensure that the package content was not tampered
```
Comment (jeanduplessis) [inline content/master/concepts/packages.md:336]:
```suggestion
tags (`vx.y.z@sha256:...`), to ensure that the package content wasn't tampered
```

File: content/v1.10/concepts/packages.md [hunk 2]
```diff
@@ -332,6 +332,10 @@ to avoid having to manually activate new versions. However, each user should
 consider their specific environment and choose a combination that makes sense
 for them.
 
+For security reasons, it's suggested using image digests instead or alongside
+tags (`vx.y.z@sha256:...`), to ensure that the package content was not tampered
```
Comment (jeanduplessis) [inline content/v1.10/concepts/packages.md:336]:
```suggestion
tags (`vx.y.z@sha256:...`), to ensure that the package content wasn't tampered
```

File: content/v1.11/concepts/packages.md [hunk 3]
```diff
@@ -332,6 +332,10 @@ to avoid having to manually activate new versions. However, each user should
 consider their specific environment and choose a combination that makes sense
 for them.
 
+For security reasons, it's suggested using image digests instead or alongside
+tags (`vx.y.z@sha256:...`), to ensure that the package content was not tampered
```
Comment (jeanduplessis) [inline content/v1.11/concepts/packages.md:336]:
```suggestion
tags (`vx.y.z@sha256:...`), to ensure that the package content wasn't tampered
```

File: content/v1.12/concepts/packages.md [hunk 4]
```diff
@@ -332,6 +332,10 @@ to avoid having to manually activate new versions. However, each user should
 consider their specific environment and choose a combination that makes sense
 for them.
 
+For security reasons, it's suggested using image digests instead or alongside
+tags (`vx.y.z@sha256:...`), to ensure that the package content was not tampered
```
Comment (jeanduplessis) [inline content/v1.12/concepts/packages.md:336]:
```suggestion
tags (`vx.y.z@sha256:...`), to ensure that the package content wasn't tampered
```

**Review by jeanduplessis — CHANGES_REQUESTED** [review]
@phisco please would you address the Vale error on line 336 of the files.

**Comment by phisco** [comment]
I tried understanding if there were errors in the lines I changed, but evidently my brain refuses to see "was not" as an actual error. Thanks @jeanduplessis .

Please hold on from merging this though, as I want a check from @AdamKorcz .

**Review by AdamKorcz — APPROVED** [review]
LGTM

**Comment by phisco** [comment]
@plumbis can you merge this? looks like I can't dismiss the requested changes by @jeanduplessis (which were actually accepted and marked as resolved 🤔)

**Pull request merged**